README.md

```diff
@@ -119,6 +119,20 @@ nodelistToArray.slice(...);
 
 The `apply` method is used to pass an array of arguments to a function with a given `this` value. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply) states that `apply` will take an array like object, which is exactly what `querySelectorAll` returns. Since we don't need to specify a value for `this` in the context of the function, we pass in `null` or `0`. The result is an actual array of DOM elements which contains all of the available array methods.
 
+In addition to the above methods, here has a new array method in ES6. `Array.from`.
```

Comment:
Might want to change to "In addition to the above method, ES6 implements a new array method,  `Array.from`."